Sources/AppcuesKit/Presentation/ExperienceRendering/ExperienceStateObserver.swift

```diff
@@ -184,19 +179,19 @@ extension ExperienceStateMachine {
 
 @available(iOS 13.0, *)
 private extension ExperienceStateObserver.StateResult {
-    var shouldTrack: Bool {
+    var shouldPublish: Bool {
         switch self {
         case let .success(state):
             return state.currentExperienceData?.published ?? false
         case let .failure(error):
-            return error.shouldTrack
+            return error.shouldPublish
         }
     }
 }
 
 @available(iOS 13.0, *)
 private extension ExperienceStateMachine.ExperienceError {
-    var shouldTrack: Bool {
+    var shouldPublish: Bool {
```

Comment:
Worth noting that now a `.noTransition` error will show up in the debugger too which would hopefully make that state machine debugging scenario easier to pick out.